packages/ui/src/kit/VError/VError.vue

```diff
@@ -0,0 +1,25 @@
+<template>
```

Comment:
can we use [this approach](https://github.com/endpass/endpass-core/blob/master/packages/ui/src/kit/VDescription/VDescription.vue)